### PR TITLE
fix integer and add task/event 'performed-by' identity relationships

### PR DESCRIPTION
**Pull request by rpiazza**

### CODE REVIEW

File: extension-definition-specifications/incident-ef7/Incident Extension Suite.adoc [hunk 1]
```diff
@@ -130,7 +130,7 @@ The values of this property *SHOULD* come from the [stixtype]#<<incident-investi
 |[stixtype]#{int_url}[integer]#
  
 |The criticality of the incident.
-If present, this value *MUST* be between 0 to 100. This can be translated into qualitative values as described in <<appendix-b,Appendix B>>.
+If present, this value *MUST* be an integer between 0 to 100. This can be translated into qualitative values as described in <<appendix-b,Appendix B>>.
```
Comment (dzbeck) [inline extension-definition-specifications/incident-ef7/Incident Extension Suite.adoc:133]: should be "between 0 and 100"

File: extension-definition-specifications/incident-ef7/Incident Extension Suite.adoc [hunk 2]
```diff
@@ -445,6 +445,11 @@ For example, a dropper running led to a ransomware tool to be downloaded and run
 |[stixtype]#{location_url}[location]#
 |The event occurred at a specific location.
 
+|[stixtype]#<<event,event>>#
+|[stixrelationship]#performed-by#
+|[stixtype]#{identity_url}[identity]#
+|The event was performed by the identity.
+
```
Comment (dzbeck) [inline extension-definition-specifications/incident-ef7/Incident Extension Suite.adoc:452]: should this be "performed" like for malware and tool objects - i.e., identity performed event?

File: extension-definition-specifications/incident-ef7/Incident Extension Suite.adoc [hunk 3]
```diff
@@ -546,7 +551,7 @@ This *MUST* match an extension that provides greater details of a specific type
 
 |*criticality* (optional)
 |[stixtype]#{int_url}[integer]#
-|The criticality of this impact. If present, this value *MUST* be between 0 to 100.
+|The criticality of this impact. If present, this value *MUST* be an integer between 0 to 100.
```
Comment (dzbeck) [inline extension-definition-specifications/incident-ef7/Incident Extension Suite.adoc:554]: "0 and 100"

File: extension-definition-specifications/incident-ef7/Incident Extension Suite.adoc [hunk 4]
```diff
@@ -1118,6 +1123,12 @@ Using these embedded relationships ensures that an incomplete sequence cannot be
 |[stixtype]#{location_url}[location]#
 |The task occurred at a specific location.
 
+
+|[stixtype]#<<task,task>>#
+|[stixrelationship]#performed-by#
+|[stixtype]#{identity_url}[identity]#
+|The task was performed by the identity.
+
```
Comment (dzbeck) [inline extension-definition-specifications/incident-ef7/Incident Extension Suite.adoc:1131]: there is an identity "performed" task relationship in Reverse Relationships section.

Comment (rpiazza) [reply extension-definition-specifications/incident-ef7/Incident Extension Suite.adoc:1131]: dropped it

File: extension-definition-specifications/incident-ef7/Incident Extension Suite.adoc (hunk 2 above)
Comment (rpiazza) [reply extension-definition-specifications/incident-ef7/Incident Extension Suite.adoc:452]: fixed

File: extension-definition-specifications/incident-ef7/Incident Extension Suite.adoc (hunk 1 above)
Comment (rpiazza) [reply extension-definition-specifications/incident-ef7/Incident Extension Suite.adoc:133]: fixed

**Comment by rpiazza** [comment]
@dzbeck - please review again :-)

File: extension-definition-specifications/incident-ef7/Incident Extension Suite.adoc (hunk 4 above)
Comment (dzbeck) [inline extension-definition-specifications/incident-ef7/Incident Extension Suite.adoc:1131]: If you have a "performed-by" relationship, should the "performed" reverse relationship be removed (see new line number 1159)?

Comment (rpiazza) [reply extension-definition-specifications/incident-ef7/Incident Extension Suite.adoc:1131]: I wanted to make them consistent.  so both are preformed and reverse relationships

**Comment by rpiazza** [comment]
@dzbeck - one last check please.  I wasn't sure what you meant about line 1159.

**Pull request merged**